src/main/java/uk/gov/hmcts/darts/event/controller/EventsController.java

```diff
@@ -168,6 +171,15 @@ public ResponseEntity<List<AdminSearchEventResponseResult>> adminSearchEvents(Ad
     @Authorisation(contextId = ANY_ENTITY_ID,
         globalAccessSecurityRoles = {SUPER_ADMIN, SUPER_USER})
     public ResponseEntity<AdminGetEventForIdResponseResult> adminGetEventById(Integer eventId) {
-        return  new ResponseEntity<>(eventService.adminGetEventById(eventId), HttpStatus.OK);
+        return new ResponseEntity<>(eventService.adminGetEventById(eventId), HttpStatus.OK);
+    }
+
+    @Override
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
+    @Authorisation(contextId = ANY_ENTITY_ID,
+        globalAccessSecurityRoles = {SUPER_ADMIN})
+    public ResponseEntity<Void> adminObfuscateEveByIds(AdminObfuscateEveByIdsRequest adminObfuscateEveByIdsRequest) {
+        this.dataAnonymisationService.obfuscateEventByIds(adminObfuscateEveByIdsRequest.getEveIds());
```

Comment:
do you need the "this" bit?

Reply:
This is just a preference thing :D

When the code gets compiled it will auto add this I just like it as it makes things more clear